hamocc/mo_sedshi.F90

```diff
@@ -59,9 +61,15 @@ subroutine sedshi(kpie,kpje,omask)
     real    :: sedlo,uebers,seddef,spresent,buried
     real    :: refill,frac
     real    :: eps=epsilon(1.)
+    real    :: acc_time=0.
 
     sedfluxb(:,:,:) = 0.
 
+    if(do_sedspinup .and. kplyear>=sedspin_yr_s .and. kplyear<=sedspin_yr_e) then
+      ! accumulated time spent due to sediment acceleration
+      acc_time = 86400.*sedspin_ncyc/31104000. ! *dtbgc/dtbgc
```

Comment:
What is the number `31104000`? A comment explaining this would be nice.

Reply:
Also, this depends on the fact that `sedshi` is called once per model day, right? Maybe it would be good to add a short comment in `hamocc4bcm` line 364 that there is a dependency on this choice in the sediment code.

Reply:
Hi @JorgSchwinger , maybe the number is incorrect? - it's a model year with 360 days in seconds - but maybe better refer to 365 days? (How is it handled in BLOM/NorESM? - naive question... - shall we maybe better introduce a parameter for this in `mo_param_bgc`?). Yes, it's due to the fact that `sedshi` is only called once per day, as far as I understand from `hamocc4bcm`.

Reply:
If the number is meant to represent seconds in a model year, then it should be `86400*365`. Is this number used more often in the code? Then it might really be worth to replace it by a parameter in `mo_param_bgc`

Reply:
It could be useful to include parameters for both 
```
seconds_in_day = 86400.
seconds_in_year = 86400.*365.
```
However, I see also references to different calendars in `phy/mod_calendar.F90`, so I'm not sure if `seconds_in_year` should be a fixed parameter.

Reply:
The calendar is defined in `phy/mod_time.F90` : ` init_timevars`. The `cesm` case is run with `noleap` = 365.
Other cases use different calendars.

Reply:
I don't think we will use something different than a 365 day calendar (at least not in the near future). One could hard code it in HAMOCC, query the BLOM calendar in hamocc_init and stop if the calendar type is not 365.

Reply:
The `fuk95` and `single_column` test cases use `360_day` calendar, so these will not work if the 365 day calendar is hard coded. We usually run these only for short tests, so maybe a warning is sufficient.

I suppose some of the input files assume a 365 day calendar. Not sure what is the best approach here, but I can see that we can run into trouble if we allow different calendar options.

Reply:
I propose to make a new issue / PR regarding calendar setting in iHAMOCC, not to bundle everything into this PR.